EmberMate/AppState.swift

```diff
@@ -40,9 +40,9 @@ class AppState: ObservableObject {
     @Published var notificationsDisabled = false
     
     @AppStorage("notifyOnTemperatureReached") var notifyOnTemperatureReached: Bool = true
-    @AppStorage("notifyOnLowBattery") var notifyOnLowBattery: Bool = true
+    @AppStorage("notifyOnLowBattery") var notifyOnLowBattery: LowBatteryLevel = .on(15)
```

Comment:
Since we're re-using the same `AppStorage` value for this, anyone migrating from `v1.7.0` is going to get a boolean value

Testing on my machine, this simply results in an empty selection, which I think might correlate to "off" but I think we could improve this a bit. Thoughts on using `notifyOnBatteryPercentage` (which would mirror my previous label suggestion)? Otherwise I'm happy to defer to you on the best way to migrate the value of an existing `@AppStorage` field

Reply:
This might be a bit more descriptive as well

```suggestion
@AppStorage("notifyOnLowBattery") var notifyOnBatteryPercentage: LowBatteryLevel = .on(15)
```

Reply:
Hm I did some testing and I was seeing old `false` values being interpreted as 0 which then gets decoded as `.off`. But I now realize I didn't test the case where it was previously set to `true` (which is `1` and which is then not one of the available levels). I'll add explicit migration code.

Reply:
I've reworked the `LowBatteryLevel` enum to just have the set of specific levels so it should hopefully never end up in the no-selection state.